From 0628f2d3bc339fc747d51340e4169d0108d19d32 Mon Sep 17 00:00:00 2001
From: chao zeng <chao.zeng@siemens.com>
Date: Thu, 16 Dec 2021 15:09:25 +0800
Subject: [PATCH 118/118] arm64: dts: ti: Indicate the green light is off when
 panic

The green light is out of control when panic occurs.
it should be off thhen

Signed-off-by: chao zeng <chao.zeng@siemens.com>
---
 arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
index 34a4bcec204a..733d6925c613 100644
--- a/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
+++ b/arch/arm64/boot/dts/ti/k3-am65-iot2050-common.dtsi
@@ -81,6 +81,7 @@ status-led-red {
 
 		status-led-green {
 			gpios = <&wkup_gpio0 24 GPIO_ACTIVE_HIGH>;
+			panic-indicator-off;
 		};
 
 		user-led1-red {
-- 
2.34.1

